test_regress/t/t_waivefile.out

```diff
@@ -0,0 +1,11 @@
+`verilator_config
+
+// TODO: Fix or keep to ignore?
```

Comment:
I think your intent here is good, but I think people will find this confusing.  
Maybe this is a good idea if we describe in the header?
Or should we not print this?
Should we print out all suppressions commented out by default?  (If so test would turn them on).

Reply:
Yeah, its probably a bit too much messing with their IDEs too :)
What do you mean by

> If so test would turn them on

Commenting out has the "disadvantage" that users would need to do something to get Verilator quiet.  It could be perceived as that we are forcing them to think about their linter warnings now, while the thing is a bit intended to push that reflection to the future and just get them rid of no-fatal or no-lint. But I may just overthink it (running replace-all "// lint_off" with "lint_off" does the job..).
I think we should at least have some introductory text. About commenting or not I will await your preference.